### ISSUE TITLE
[TF2] [URGENT!!!!] A new Hack called "Chillware" has been found. Please fix this!

### ISSUE THREAD

**Issue by VirtualSoda** [ValveSoftware/Source-1-Games#2131]
It seems there is a new hack called "chillware" It's deemed as not detectable https://www.youtube.com/watch?v=ibwhsJwQZZE Here's a video where you can see people use the hack in a casual match. 

Please fix this!

**Comment by hahagoni** [comment]
Suspected creator of cheat. [No surprise it's undetected as it's a private software with smaller distribution](https://steamcommunity.com/profiles/76561198346386083)
Therefore, this shouldn't affect many users yet.
The main issue appears to be being able to shoot through solid (blockbullet, nodraw) objects.

**Comment by agrastiOs** [comment]
Valve, you could buy the hack from the creator for two keys, manually ban him, and make the hack detectable.

**Comment by Wasmanr2** [comment]
Using an fake account to buy that would be smarter. (I mean who the hell will sell you as a valve member a hack software ?)

**Comment by agrastiOs** [comment]
Yes, of course. I didn't mention that, because come on ; it's common sense.
